src/electronApp/angularApp/components/status/status.component.ts

```diff
@@ -115,14 +145,33 @@ export class StatusComponent implements OnInit {
   /**
    * Disconnects from the printer.
    */
-  public Disconnect(): void{
+  public Disconnect(): void {
     this.printerService.Disconnect();
```

Comment:
As you mentioned in the PR description, I think it would be good to have a confirmation dialog here. You can see an example of how to do this [here](https://github.com/andycb/AdventurerClientJS/blob/c2fd06abb3a7551ed0bb4ad1be9271d6d4bf9978/src/electronApp/angularApp/components/app/app.component.ts#L44) and [here](https://github.com/andycb/AdventurerClientJS/tree/master/src/electronApp/angularApp/components/connection-error-dialog)

Reply:
Added the Stop Printing confirmation with 4f8f329727431eeb48fc7a3d2fe70a63a393d055 and fixed some mistakes in ba30c8bfaa1a1ea3bce1db5ca0b09ec484f81a0e and 6781b342ec4988d9dbbe3c850944f3705c7b89f4.